backend/YouTubeV2.Application/Model/User.cs

```diff
@@ -8,6 +8,8 @@ public class User : IdentityUser
         public string Name { get; init; }
         public string Surname { get; init; }
 
+        public virtual ICollection<Subscription> Subscriptions { get; init; }
```

Comment:
Instead of this I think you could add ICollection<User> Subscriptions. I think it would make things simpler but it's just a suggestions and this way works too.

Reply:
The database would look the same only c# would change

Reply:
Checked this approach, however it requires much more work with the same result and as such I stayed with this one